doc/10-icinga-template-library.md

```diff
@@ -3278,6 +3278,146 @@ printer_health_oids		| **Optional.** A list of oids which are downloaded and wri
+
+##### thola-ups <a id="plugin-contrib-command-thola-ups"></a>
+
+Checks whether a UPS device has its main voltage applied.
+
+Custom variables passed as [command parameters](03-monitoring-basics.md#command-passing-parameters):
+
+Name                                        | Description
+--------------------------------------------|-----------------------------------------------------------------
+thola_api_address                           | **Required.** Address of the Thola API to connect to
+thola_device_address                        | **Required.** The host's address. Defaults to "$address$" if the host's address attribute is set, “$address6$” otherwise
+thola_device_snmp_community                 | **Optional.** SNMP community of the device
+thola_ups_batt_current_critical_max         | **Optional.** High critical threshold for the battery current
+thola_ups_batt_current_critical_min         | **Optional.** Low critical threshold for the battery current
+thola_ups_batt_current_warning_max          | **Optional.** High warning threshold for the battery current
+thola_ups_batt_current_warning_min          | **Optional.** Low warning threshold for the battery current
+thola_ups_batt_temperature_critical_max     | **Optional.** High critical threshold for the battery temperature
+thola_ups_batt_temperature_critical_min     | **Optional.** Low critical threshold for the battery temperature
+thola_ups_batt_temperature_warning_max      | **Optional.** High warning threshold for the battery temperature
+thola_ups_batt_temperature_warning_min      | **Optional.** Low warning threshold for the battery temperature
+thola_ups_current_load_critical_max         | **Optional.** High critical threshold for the current load
+thola_ups_current_load_critical_min         | **Optional.** Low critical threshold for the current load
+thola_ups_current_load_warning_max          | **Optional.** High warning threshold for the current load
+thola_ups_current_load_warning_min          | **Optional.** Low warning threshold for the current load
+thola_ups_rectifier_current_critical_max    | **Optional.** High critical threshold for the current rectifier
+thola_ups_rectifier_current_critical_min    | **Optional.** Low critical threshold for the current rectifier
+thola_ups_rectifier_current_warning_max     | **Optional.** High warning threshold for the current rectifier
+thola_ups_rectifier_current_warning_min     | **Optional.** Low warning threshold for the current rectifier
+thola_ups_system_voltage_critical_max       | **Optional.** High critical threshold for the system voltage
+thola_ups_system_voltage_critical_min       | **Optional.** Low critical threshold for the system voltage
+thola_ups_system_voltage_warning_max        | **Optional.** High warning threshold for the system voltage
+thola_ups_system_voltage_warning_min        | **Optional.** Low warning threshold for the system voltage
```

Comment:
... like these. Do you know what I mean? Btw. consider just copying them from here.